tests/ui/needless_return.stderr

```diff
@@ -136,10 +136,10 @@ LL | |             return;
    = help: remove `return`
 
 error: unneeded `return` statement
-  --> $DIR/needless_return.rs:98:14
+  --> $DIR/needless_return.rs:98:13
    |
 LL |         _ => return,
-   |              ^^^^^^
```

Comment:
I'm a little bit confused, that the changes above seem to have changed the span of the lint. Do you know where this comes from?

I'm guessing that these cases were marked as `RetReplacement::Empty` before while they're now `RetReplacement::Unit`. That is totally fine, but it would be good if the span stays the same, or the suggestion at least also includes a leading space :)

Reply:
Yeah I also was confused with that and even asked on zulip what maybe the possible reason for it 🤔

Reply:
Fixed this by changing the default we send xD